Runtime/Launcher.cs

```diff
@@ -33,7 +35,8 @@ internal static void ResetEventHandlers()
         /// Run autopilot
         /// </summary>
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.AfterSceneLoad)]
-        public static void Run()
+        // ReSharper disable once Unity.IncorrectMethodSignature
+        public static async UniTask Run()
```

Comment:
![nits](https://img.shields.io/badge/review-nits-green.svg)

Maybe you can use `UniTaskVoid`.

Reply:
Fixed in d00d6a0390d0a6141da00f65870dba4fa18f4ea6.